### ISSUE TITLE
usage with webpack/browserify

### ISSUE THREAD

**Issue by tnrich** [alertifyjs/alertify.js#62]
Hey, so it looks like this can be installed via npm, but it doesn't look like it can be consumed using webpack/browserify.

eg. 

`import alertify from 'alertify.js';`

doesn't work. 

Any reason why this should be the case?

**Comment by bradberger** [comment]
That's a good question, honestly I've never used it that way, so it might be a mistake. We just had a recent PR from @sakabako 944ba40bf45d9e070b4c35d37de3a9b21a8512f2 that might have addressed this already, but really I'm not sure. 

Could you check it out and let me know if that's the solution we need?

**Comment by sakabako** [comment]
try `import default from 'alertify.js'`

**Comment by tnrich** [comment]
`import alertify from 'alertify.js';`

did not work for me so I think it is broken. 

And it looks like the problem is that this change didn't make it into the npm module yet: 
```js 
// AMD, window, and NPM support
    if ("undefined" !== typeof module && !! module && !! module.exports) {
        module.exports = Alertify;
    } else if (typeof define === "function" && define.amd) {
        define(function() {
            return new Alertify();
        });
    } else {
        window.alertify = new Alertify();
    }
```

You probably just need to publish the new code to npm. 

Thanks for the response :)

**Comment by bradberger** [comment]
Okay, perfect. Thanks for the update. I've just got to bump the version and then I'll publish it.

**Comment by tnrich** [comment]
Great thanks!

**Comment by bradberger** [comment]
It's published now as `1.0.7`. Feel free to re-open if that doesn't fix things.
